src/Rel_PT_Emission_Profile.jl

```diff
@@ -0,0 +1,73 @@
+#imports
+using Plots
+
+#functions
+function r_isco(a_star, M)
+    """
+    From Reynolds (2020) (Eq2-4)
+    Calculates the radius of the inner most stable circular orbit of the a black hole,
+    with spin a_star, and mass M.
+    """
```

Comment:
Docstrings in Julia go before the function, i.e.
```julia
"""
    function r_isco(a_star, M)

From Reynolds (2020) (Eq2-4)
Calculates the radius of the inner most stable circular orbit of the a black hole,
with spin a_star, and mass M.
"""
function r_isco(a_star, M)
    # ...
end
```

What's also pretty nice is the docstrings can be full markdown text, with LaTeX functions in them too -- check out [this code from GeodesicTracer.jl](https://github.com/astro-group-bristol/GeodesicTracer.jl/blob/0225548b90655cac2491024b6d1ea05d3100e11d/src/coordinates.jl#L2-L11), which renders into [this documentation](http://www.star.bris.ac.uk/fbaker/docs/GeodesicTracer.jl/first_order_integration/#GeodesicTracer.%CE%A3).

We'll set up auto-documentation later down the line! :)